ideascube/templates/ideascube/includes/cards/bouquineux.html

```diff
@@ -0,0 +1,5 @@
+{% load i18n %}
+<div class="card tinted bouquineux">                                                                                                                                                          
+  <h4 class="flow"><span class="theme learn">{% trans 'info' %}</span><span><a href="http://kiwix.koombook.lan/bouquineux_07_2013/">{% trans "Bouquineux" %}</a></span></h4>               
+  <p>{% trans "Bibliothèque de livres numériques" %}</p>                                                                                                                                           
```

Comment:
We should either have the string in English or not translate it. Same for other strings around.